R/plotCCA.R

```diff
@@ -123,6 +123,7 @@
 #'     )
 #'
 #' # Create RDA plot coloured by variable
+#' suppressWarnings({
```

Comment:
Piping might look better, i.e.,

function() |>
    # Remove this line to see warnings
    suppressWarnings()

It is also easier to remove

Reply:
My mistake. I did not realize that you suppressed whole chunk once. That is good solution.